packages/alpine.rb

```diff
@@ -0,0 +1,22 @@
+require 'package'
+
+class Alpine < Package                 # The first character of the class name must be upper case
+  description 'An easy-to-use email client.'
```

Comment:
Consider using a more descriptive description. We often just use the description from NetBSD's pkgsrc or Arch Linux's PKGBUILDs. Arch describes this package as: "The continuation of the Alpine email client from University of Washington"